wdl-analysis/Cargo.toml

```diff
@@ -12,6 +12,7 @@ documentation = "https://docs.rs/wdl-analysis"
 
 [dependencies]
 wdl-ast = { path = "../wdl-ast", version = "0.9.0" }
+wdl-lint = { path = "../wdl-lint", version = "0.8.0" }
```

Comment:
So this dependency will prevent us from basing `wdl-lint` on top of `wdl-analysis` in the future.

I was hoping that we could expose these functions in the `wdl` crate itself so that the upstream crates didn't have potentially unnecessary inter-dependencies like this.